build_debian.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
I'm not super keen on this being in the root. It looks like a docker thing anyway - can it go in the docker dir?

(and can it have a comment saying what it does?)